request.go

```diff
@@ -31,25 +31,31 @@ type Config struct {
 	WaitHttp           time.Duration
 	HTTPRequestTimeout time.Duration
 
+	PubSubServer string
+
 	StorageHostServer    []string
 	StorageDB            int
 	TempStorageKeyPrefix string
 	ExpiryTime           time.Duration
 	StorageTimeout       time.Duration
+	Channel              string
 }
 
-type Requestor interface {
-	SendRequest(ctx context.Context, url, action string, payload []byte, header map[string]string, key string) (statusCode int, responseBody []byte, err error)
-}
+//type Requestor interface {
+//	SendRequest(ctx context.Context, url, action string, payload []byte, header map[string]string, key string) (statusCode int, responseBody []byte, err error)
+//}
 
 // Client will handle http request response by extending go http.Client package
 type Client struct {
 	HTTPClient         *http.Client
 	CacheClient        cache.Cacher
+	PubsubClient       cache.Cacher
 	ExpiryTime         time.Duration
 	MainTimeOut        time.Duration
 	WaitHttp           time.Duration
 	HTTPRequestTimeout time.Duration
+	Channel            string
```

Comment:
Should the user create a channel manually? or let the library use a default channel?

Reply:
in this case i let the user create the channel manualy. to prevent same usage of channel for different host. wdyt

Reply:
Maybe we can set a defaultChannel instead. for instead, use {uuid}_lazyhttp_retry

Reply:
NVM lets do it later.